sdk/storage/azure-storage-file-share/tests/test_file.py

```diff
@@ -2089,17 +2381,20 @@ def test_sas_signed_identifier(self, storage_account_name, storage_account_key):
         content = file_client.download_file().readall()
 
         # Assert
-        self.assertEqual(self.short_byte_data, content)
+        assert self.short_byte_data == content
+
+        return variables
 
+    @pytest.mark.live_test_only
```

Comment:
Since you are recording most other SAS tests here, any reason this one is live only? Same with some of the "shared access" ones below. It's fine if the reason is you didn't want to make all the SAS tests recorded since we're not really requiring it as part of this migration.